pallets/thea-message-handler/src/lib.rs

```diff
@@ -232,6 +244,21 @@ pub mod pallet {
 			<IncomingNonce<T>>::put(nonce);
 			Ok(())
 		}
+
+		/// A governance endpoint to update outgoing nonces
+		#[pallet::call_index(3)]
+		#[pallet::weight(Weight::default())]
```

Comment:
Should be added  benchmarks and weights?

Reply:
@zktony missed the benchmark for that extrinsic but it is technically same as update incoming nonce, so use it's weight for now.